keel-sqs/src/test/kotlin/com/netflix/spinnaker/keel/sqs/SqsIntegrationTests.kt

```diff
@@ -0,0 +1,60 @@
+package com.netflix.spinnaker.keel.sqs
+
+import com.netflix.spinnaker.config.SqsConfiguration
+import com.netflix.spinnaker.keel.annealing.ResourceActuator
+import com.netflix.spinnaker.keel.annealing.ResourceCheckQueue
+import com.netflix.spinnaker.keel.api.ResourceName
+import com.netflix.spinnaker.keel.api.SPINNAKER_API_V1
+import com.nhaarman.mockitokotlin2.doAnswer
+import com.nhaarman.mockitokotlin2.stub
+import com.nhaarman.mockitokotlin2.verify
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment.NONE
+import org.springframework.boot.test.mock.mockito.MockBean
+import strikt.api.Assertion
+import strikt.api.expectThat
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.TimeUnit.SECONDS
+
+@SpringBootTest(
+  classes = [LocalStackSqsConfiguration::class, SqsConfiguration::class],
+  properties = [
+    "sqs.enabled=true",
+    "sqs.account-name=test",
+    "sqs.queue-arn=arn:aws:sqs:ap-south-1:530822479253:keel-test-resource_check"
+  ],
+  webEnvironment = NONE
+)
+internal class SqsIntegrationTests {
```

Comment:
Integration test that uses localstack to stand up an SQS API and make sure the queue read / write work in congress.

Thoughts on whether this has value? It's pretty slow due to localstack pulling down a fucking giant Docker image.

Reply:
I don't feel like it has a ton of value. What cases would this docker image version of SQS catch that we wouldn't catch another way? Seems like increased build time is a win regardless.